session_0/word_processing.rb

```diff
@@ -34,11 +46,28 @@ def words_with_prefix(prefix, words)
 # The function `similarity_score` takes two words and returns the
 # similarity score (an integer).
 def similarity_score(word_1, word_2)
```

Comment:
`similarity_score('apple', 'apple')` will be stuck forever - try and see why

Reply:
Ohhh got it thank you

Reply:
Hey, so could you please explain in items.rb, why do we need an extra instance method for quantity, when there already is an instance variable quantity, and it is also under attr_accessor?

Reply:
We don't need an extra instance method - why do you feel the need for it?

Reply:
Yeah, I realized that I could include it in attr_accessor